packages/gestalt/src/Pulsar.css.flow

```diff
@@ -1,8 +1,8 @@
 // @flow strict
 /* This file is automatically generated by css-modules-flow-types */
 declare module.exports: {|
-  +'AppearAnimation': string,
```

Comment:
@dangerismycat since can be imported in JavaScript, wouldn't it be better to avoid kebab-case?

E.g. `styles.AppearAnimation` instead of `styles['appear-animation']`

Reply:
Hm…good point. I kept kebab-case as most existing usages were in that case, and I was hesitant to change the animations related to the searchbox since I think that relies on matching some code we don't own. I hadn't considered importing in to JS. Do we do that with animations currently? Do you expect we'll need to in the future?

Reply:
@dangerismycat we probably don't right now but I do expect it in the future - especially if we start to become more animation heavy in gestallt

Reply:
I'm going to go ahead and merge this PR now, and we can revisit the lint rule if necessary in the future. I don't have strong feelings about it at all.